kafka-client-bootstrap/build.gradle.kts

```diff
@@ -15,6 +15,10 @@ plugins {
     id("responsive.docker")
 }
 
+repositories {
```

Comment:
Had to put this into all the build.gradle files so I could compile from my local AK branch. I can remove all of these before merging if anyone feels strongly about it, but if it doesn't do any harm then I'd rather keep these for the next time I'm working off of a dev branch